src/isomorphism/mod.rs

```diff
@@ -50,13 +50,17 @@ use pyo3::Python;
 /// :param bool id_order: If set to ``False`` this function will use a
 ///     heuristic matching order based on [VF2]_ paper. Otherwise it will
 ///     default to matching the nodes in order specified by their ids.
+/// :param int call_limit: An optional bound on the number of states that VF2 algorithm
+///     visits while searching for a solution. If it exceeds this limit, the algorithm
+///     will stop and return ``False``. Default: ``None``.
 ///
 /// :returns: ``True`` if the 2 graphs are isomorphic ``False`` if they are
 ///     not.
 /// :rtype: bool
-#[pyfunction(id_order = "true")]
+#[pyfunction(id_order = "true", call_limit = "None")]
```

Comment:
You don't actually need this as it's implicit with the `Option<usize>` type, the pyo3 pyfunction macro will default it to `None` so we don't need to explicitly set the default for the argument (it's the reason `node_matcher` and `edge_matcher` aren't in the list here.

```suggestion
#[pyfunction(id_order = "true")]
```